src/lib/components/Header.svelte

```diff
@@ -1,15 +1,18 @@
 <script lang="ts">
 	import ProfileButtonHeaderEmbed from '$lib/components/ProfileButton/ProfileButtonHeaderEmbed.svelte';
 
-	let { title }: { title: string } = $props();
+	let { title, info }: { title: string; info?: string | null } = $props();
 </script>
 
 <div class="navbar bg-base-200 px-4 py-2 rounded-2xl shadow-lg gap-2">
 	<label for="my-drawer" class="btn btn-square btn-ghost drawer-button lg:hidden">
 		<i class="fa-solid fa-bars text-xl"></i>
 	</label>
-	<div class="grow">
+	<div class="grow flex flex-col items-start">
 		<h1 class="lg:text-2xl lg:font-light">{title}</h1>
+		{#if info && info.length > 0}
```

Comment:
Why do you display this inside the Navbar? Maybe I am missunderstanding what you aimed for with the info field, but displaying blocks of text does not seem like a good idea inside the navbar :D

Reply:
Ahh didn't get the memo of this being a navbar, thought thats a general "Header" component :D
Where would this be applicable? Alternatively I'd put this directly into the page component then?

Reply:
Yeah, thats how I pictured it. I was thinking maybe implement this on each page here (see picture) with a component, maybe with a [collapsable](https://daisyui.com/components/collapse/) and by default only a preview of the information? Or a [tabs setup](https://daisyui.com/components/tab/) with e.g. the delegation members would also be an option I think.

![CleanShot 2024-10-11 at 11 25 43@2x](https://github.com/user-attachments/assets/06cb7793-6c3c-42e4-8f61-e996c1c130e6)

Reply:
I see, I'd probably put it in the layout since it doesn't change throughout the dashboard states but I'll think of something. Thanks for the hint!

Reply:
What do you think about this?
![image](https://github.com/user-attachments/assets/93fc7226-69d1-4d4f-9e77-32ddf28bac85)
Displaying it above the dynamic content allows for DRY and removes the necessity to embed an info component into the sub components on the dashboard page

Reply:
Looks good. If it is collapsed by default so it does not draw attention await from the more important blocks below, it would be okay for me. Maybe also give it a more formal Headline like "Allgemeine Informationen"? Also, I really would add a preview when collapsed, since it is otherwise impossible to notice changes that one should be made aware of. I mean having a collapsed info box always comes with this problem, but having the first 2-3 Lines of text would make it at least possible for editors to state new additions in the first sentence of the box...

Reply:
Now that I am thinking of it: Maybe a "new" Badge would be a good idea for that reason. Save the current to local storage and compare it to what is fetched maybe?

Reply:
I like the "new" badge idea! I'll give that a shot!